src/ophyd_async/epics/adcore/__init__.py

```diff
@@ -0,0 +1,38 @@
+from ._ad_base import (
+    DEFAULT_GOOD_STATES,
+    ADBase,
+    ADBaseShapeProvider,
+    DetectorState,
+    set_exposure_time_and_acquire_period_if_supplied,
+    start_acquiring_driver_and_ensure_status,
+)
+from ._hdf_writer import HDFWriter
+from ._nd_file_hdf import NDFileHDF
+from ._nd_plugin import ADBaseDataType, NDPluginStats
```

Comment:
Should: be IO classes
```suggestion
from ._nd_file_hdf import NDFileHDFIO
from ._nd_plugin import ADBaseDataType, NDPluginStatsIO
```

Reply:
Renamed both.